react/webpack.config.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
       { test: /\.scss$/, include: path.normalize(__dirname + '/scss/person-graph.scss'), loader: 'style!css!autoprefixer!sass' },
       { test: /\.scss$/, include: path.normalize(__dirname + '/scss/composers-app.scss'), loader: 'style!css!autoprefixer!sass' },
       { test: /\.scss$/, include: path.normalize(__dirname + '/scss/relations-app.scss'), loader: 'style!css!autoprefixer!sass' },
+      { test: /\.scss$/, include: path.normalize(__dirname + '/scss/feedback-component.scss'), loader: 'style!css!autoprefixer!sass' },
```

Comment:
This is not a feedback to this particular PR, but could you replace all those scss include lines with a single include of the whole `scss` folder?

`{ test: /\.scss$/, include: path.normalize(__dirname + '/scss/'), loader: 'style!css!autoprefixer!sass' },`